andela/savage_rangers-ah
# #166240808 Integrate Travis CI

#### What does this PR do?
Have Travis CI integrated with the main repository

#### Description of Task to be completed?
add travis.yml file with the instruction included.

#### How should this be manually tested?
Head over to savage_rangers-ah repo and check out the badge it is failing or passing.

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
 #166240808
